features/step_definitions/element_steps.rb

```diff
@@ -87,13 +95,18 @@
   @test_site.home.remove_container_with_element_btn.click
 end
 
-Then('I receive an error when a section with the element I am waiting for is removed') do
-  expect { @test_site.home.container_with_element.wait_until_embedded_element_invisible }
-    .to raise_error(Capybara::ElementNotFound)
+Then(
```

Comment:
I don't like this one atm. Can we just revert it and have a re-think later (Leaving rubocop rules inplace if needs be)